frontend/src/components/Customer/TrackOrder.tsx

```diff
@@ -60,7 +77,23 @@ const invoiceSubtotal = subtotal(rows);
 const invoiceTaxes = TAX_RATE * invoiceSubtotal;
 const invoiceTotal = invoiceTaxes + invoiceSubtotal;
 
-function OrderItemsTable() {
+export interface Order {
```

Comment:
This is copy-pasted from the Employees/Order.tsx file, with comment. Import the interface instead, and remove redundant comments.